README.md

```diff
@@ -299,6 +295,36 @@ runWhenIdle(() => {
 A detailed explanation of why `AsyncContext.Snapshot` is a requirement can be
 found in [SNAPSHOT.md](./SNAPSHOT.md).
 
+### `AsyncContext.Snapshot.wrap`
```

Comment:
It'd be great if these docs included a use case which made sense (e.g., userspace queuing/batching, where it's often more convenient to do `wrap` rather than using the `Snapshot` API).